generators/client/templates/angular/src/main/webapp/app/shared/util/datepicker-adapter.ts

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
Rename this file to have a `_` before the name

Reply:
You also need to add all three of these files to https://github.com/jhipster/generator-jhipster/blob/master/generators/client/files-angular.js#L410-L425

Reply:
I think its better to move the general util pipes to ng-jhipster lib, rather than generate them

Reply:
That what I said in the main comment. But the idea to use angular2 moment is even better (if it works well with angular 5+